src/Cimpress.Nancy/project.json

```diff
@@ -27,7 +26,8 @@
     "netstandard1.6": {
       "imports": "dnxcore50",
       "dependencies": {
-        "NETStandard.Library": "1.6.0"
+        "NETStandard.Library": "1.6.0",
+        "System.Net.Requests": "4.0.11"
```

Comment:
Yes, System.Net is a dependency of AWSSDK.Core so this had to be re-introduced in order to catch exceptions in OnError